### DISCUSSION TITLE
Events in Field, Chip components

### DISCUSSION THREAD

**Discussion by futornoi** [material-components/material-web#5216] [Q&A]
Hi everyone. I'm currently trying to create m3 components in React's own format using the lit library. I researched every element I needed and had some problems. I am facing irrelevant documentation (looking on GitHub, maybe somewhere else). 
I am currently checking chip and text field components and have not found documentation for events (change, input for fields, and any event on the delete chip button). Any support would be appreciated

**Comment by datvm** [comment]
See #5118 . You will want to check for `click` event and `remove` event. Note that `remove` event does not bubble so you cannot rely on event bubbling (you can still use [capture](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener)). I wrote a custom JS workaround to bubble the event when implementing it for Blazor, see #4951 .
